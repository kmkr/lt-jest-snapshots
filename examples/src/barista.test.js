import barista from './barista.js'

describe('the barista', () => {

    it('should make a coffee correctly', () => {
        const coffee = barista({
            type: 'coffee',
            centiliters: 50,
        });
        expect(coffee).toMatchSnapshot();        
    });

});