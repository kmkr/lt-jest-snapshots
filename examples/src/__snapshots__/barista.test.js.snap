// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the barista should make a coffee correctly 1`] = `
Object {
  "cup": "medium-cup",
  "type": "coffee",
}
`;
